white_label/brands/mabulle/android/app/src/main/res/values/base_background.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="base_background">#0065FD</color>

```

Comment:
Where does this color comes from? Shouldn't this be the Primary color? (`#0066FF` for MaBulle)

Reply:
That was the color I got on Figma, maybe got an issue when translating from HSL to hex, using your value and removing this file, using colors.xml instead